CODE_OF_CONDUCT.md

```diff
@@ -1,46 +1,9 @@
-# Contributor Covenant Code of Conduct
```

Comment:
This is not a Covenant in the sense of 

"a formal alliance or agreement made by God with a religious community or with humanity in general", nor is it in the sens of law:

"a promise to engage in or refrain from a specified action (including restrictive covenant, a restriction on the use of property)."

It is an assumption, and an appeal to authority at that.

The scope of creating and fostering community is in some places not misplaced, but as such should then not be limited to contributors, as they are only part thereof, and at that not be an assumption by its introduction or as a result its utilization.

More importantly: Why?